pkg/resolver/resolver.go

```diff
@@ -0,0 +1,21 @@
+package resolver
+
+import (
+	"net"
+	"time"
+)
+
+func IsHostResolvable(host string) bool {
+	timeout := time.After(200 * time.Millisecond)
```

Comment:
This is 200ms

Reply:
Resolved: https://github.com/narvikd/nubedb/pull/10/commits/a0f315b5347c14c25d7f560d7b49e3866fc49374